class CommandError(Exception):
    pass


class PrefixError(Exception):
    pass
